libguard/guard_log.cpp

```diff
@@ -5,29 +5,70 @@
 
 #include <iostream>
 
+
 namespace openpower
 {
 namespace guard
 {
 namespace log
 {
+struct guard_log_settings 
+{
+	int log_level;
+	guard_log_func_t log_func;
+	void *log_func_priv_data;
+};
+
+static void guard_log_default(void *priv, const char *fmt, va_list ap)
+{
+  (void)(priv);
```

Comment:
any specific reason for this code?

Reply:
if not we get a warning of unused *priv, which lands into error

Reply:
FYI, we could add a comment to that variable in the function arguments list.

For example, `static void guard_log_default(void /* *priv */, const char *fmt, va_list ap)` and documentation, we can mentioned it as "unused parameter".